Makefile

```diff
@@ -31,6 +31,11 @@ horologium:
 	oc process -f cluster/ci/config/prow/openshift/horologium.yaml | oc apply -f -
 .PHONY: horologium
 
+jenkins-proxy:
```

Comment:
secrets?

Reply:
Yeah, missing secrets and config. Will add in a new push.